src/test/java/redis/clients/jedis/tests/ConnectionTest.java

```diff
@@ -1,17 +1,23 @@
 package redis.clients.jedis.tests;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
-
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.util.LinkedList;
+import java.util.List;
+
 import redis.clients.jedis.Connection;
+import redis.clients.jedis.JedisCommandListener;
 import redis.clients.jedis.Protocol.Command;
 import redis.clients.jedis.commands.ProtocolCommand;
 import redis.clients.jedis.exceptions.JedisConnectionException;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
 public class ConnectionTest {
```

Comment:
Connection operation or connection methods are not going to be tested. Move the test methods in a new test class.